Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -69,9 +69,12 @@ void VideoConfig::Load(const std::string& ini_file)
   settings->Get("OverlayStats", &bOverlayStats, false);
   settings->Get("OverlayProjStats", &bOverlayProjStats, false);
   settings->Get("DumpTextures", &bDumpTextures, false);
-  settings->Get("HiresTextures", &bHiresTextures, false);
-  settings->Get("ConvertHiresTextures", &bConvertHiresTextures, false);
-  settings->Get("CacheHiresTextures", &bCacheHiresTextures, false);
+  if (!settings->Get("CustomTextures", &bCustomTextures, false))
+  {
+	  settings->Get("HiresTextures", &bCustomTextures, false);
+  }
```

Comment:
Well all the code I see in VisualStudio looks as I expected it to. Maybe it auto-adds the look even if there's no actual tab in the raw files.